kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -173,7 +174,10 @@ public boolean containsDataInTimeRange(long startTimeMs, long endTimeMs) {
     ensureTrue(
         endTimeMs - startTimeMs >= 0,
         String.format(
-            "end timestamp %d can't be less than the start timestamp %d.", endTimeMs, startTimeMs));
+            Locale.ROOT,
```

Comment:
What are the advantages of using Locale.ROOT over leaving it null and let the system pick the default?

Reply:
That's a good question

As per the jdocs

```
Useful constant for the root locale. The root locale is the locale whose language, country, and variant are empty ("") strings. This is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.
Since:
1.6
```

and if we don't provide a locale `The locale used is the default locale for formatting for this instance of the Java virtual machine.` will get picked

So unsure, but this is the convention that lucene/solr and other projects follow so I kinda went along